cmd/clusterctl/cmd/init.go

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+type initOptions struct {
+	kubeconfig              string
+	coreProvider            string
+	bootstrapProviders      []string
+	infrastructureProviders []string
+	targetNamespace         string
+	watchingNamespace       string
+	force                   bool
+}
+
+var io = &initOptions{}
+
+var initCmd = &cobra.Command{
+	Use:   "init",
+	Short: "Initialize a management cluster for Cluster API",
+	Long: LongDesc(`
+		Initialize a management cluster for Cluster API by installing cluster API core components,
+		the kubeadm bootstrap provider, and the selected bootstrap and infrastructure providers.
+
+		The management cluster should be an existing Kubernetes cluster, and the identity used
+		for accessing the cluster must have enough privileges for installing cluster API providers.
+
+		Use 'clusterctl config providers' to get the list of available providers; if necessary, edit
+		the .clusterctl file to add new provider configurations or to customize existing ones.
```

Comment:
s/.cluster/.clusterctl.yaml -- assuming this is a typo

Reply:
done